ethereumj-core/src/main/java/org/ethereum/sharding/crypto/Sign.java

```diff
@@ -36,24 +44,30 @@
     BigInteger aggPubs(List<BigInteger> pubKeys);
 
     class Signature {
-        public BigInteger r;
-        public BigInteger s;
+        public BigInteger value;
```

Comment:
It doesn't look to me that having `Signature` class is something useful. Maybe turn it into `byte[]`?